clients/test/lightrider-entry-test.js

```diff
@@ -14,7 +14,10 @@ const LHError = require('../../lighthouse-core/lib/lh-error.js');
 describe('lightrider-entry', () => {
   describe('#runLighthouseInLR', () => {
     it('returns a runtimeError LHR when lighthouse throws a runtimeError', async () => {
-      const connectionError = new LHError(LHError.errors.FAILED_DOCUMENT_REQUEST);
+      const connectionError = new LHError(
+        LHError.errors.FAILED_DOCUMENT_REQUEST,
+        {errorDetails: 'Bad connection req'}
+      );
```

Comment:
I think that is what the includes checks, that the runtime message in the end is the same as the orig friendly msg post a round of translation since it is outside of the Runner.run which translates errors before it exits.  It uses the includes instead of equal b/c LR appends the [msg to the friendly msg](https://github.com/GoogleChrome/lighthouse/blob/3b562abb0525ad0605d7cfc754e5f0472fb39d19/clients/lightrider-entry.js#L69) which may not be desired?  It's the first I'm really seeing it.

Reply:
I had noticed that `runLighthouseInLR` doesn't run i18n on its generated output, hence the question, but the new `err.friendlyMessage = i18n.getFormatted(err.friendlyMessage, settings.locale);` line in `runner.js` translates the error `friendlyMessage` before it gets back to `runLighthouseInLR`, so localization is good to go!